invenio_app_rdm/records_ui/views/decorators.py

```diff
@@ -191,18 +201,23 @@ def view(**kwargs):
             if is_preview:
                 try:
                     files = draft_files_service().list_files(**read_kwargs)
+                    media_files = draft_media_files_service().list_files(**read_kwargs)
                 except NoResultFound:
                     files = files_service().list_files(**read_kwargs)
+                    media_files = media_files_service().list_files(**read_kwargs)
             else:
                 files = files_service().list_files(**read_kwargs)
+                media_files = media_files_service().list_files(**read_kwargs)
 
             kwargs["files"] = files
+            kwargs["media_files"] = media_files
```

Comment:
could be split potentially to a separate decorator, but I didn't see the need for the time being